src/network/eventloop.rs

```diff
@@ -121,7 +190,7 @@ impl EventLoop {
                 }
             },
             SwarmEvent::Behaviour(ComposedEvent::RequestResponse(
-                RequestResponseEvent::OutboundFailure {
```

Comment:
Are these just type aliases for legibility when used without the module?

Reply:
libp2p is removing them essentially.

Reply:
so these alias's no longer exist, unless we wanted to create ourselves.